### PR TITLE
Fix repository cretaion path on publisher

### PR DESCRIPTION
**Pull request by ahmam**
Fix repository cretaion path on publisher 
Fix  copy cert path

### CODE REVIEW

File: tasks/publisher.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@
 - name:  Create the repository available for writing publisher
   ansible.builtin.command: >-
     /usr/bin/cvmfs_server mkfs -w http://{{ item.gateway }}/cvmfs/{{ item.repository }} 
-      -u gw,/srv/cvmfs/{{ item.repository }},http://{{ item.gateway }}:{{ item.gateway_port | default('4929') }}/api/v1
+      -u gw,/srv/cvmfs/{{ item.repository }}/data/txn,http://{{ item.gateway }}:{{ item.gateway_port | default('4929') }}/api/v1
```
Comment (btravouillon) [inline tasks/publisher.yml:53]: Did this path change recently? What caused the regression?

File: tasks/publisher.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 - name: Copy certificate key on publisher 
   ansible.builtin.copy:
     content: "{{ item.crt }}"
-    dest: /var/tmp/keys/mycert.crt
+    dest: /var/tmp/keys/"{{ item.crt }}"
```
Comment (btravouillon) [inline tasks/publisher.yml]: Is `item.crt` both the content of the file and the file name?

File: tasks/publisher.yml (hunk 1 above)
Comment (ahmam) [reply tasks/publisher.yml:53]: this path has not changed but it is an omission of the rest of the path .
`sudo cvmfs_server mkfs -w http://$S0_IP/cvmfs/repo.organization.tld \
                      -u gw,/srv/cvmfs/repo.organization.tld/data/txn,http://$S0_IP:4929/api/v1 \
                      -k /path/to/keys/dir -o $USER repo.organization.tld
`

File: tasks/publisher.yml (hunk 2 above)
Comment (ahmam) [reply tasks/publisher.yml]: My bad it's the name of repository i already fixed it.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**